src/commands/app/deploy.js

```diff
@@ -100,7 +101,10 @@ class Deploy extends BuildCommand {
       // - break into smaller pieces deploy, allowing to first deploy all actions then all web assets
       for (let i = 0; i < keys.length; ++i) {
         const k = keys[i]
-        const v = values[i]
+        let v = values[i]
+
+        v = setRuntimeApiHostAndAuthhandler(v)
```

Comment:
does v ever change from this point on?
I would do: `const v = setRuntimeApiHostAndAuthHandler(values[i])`